src/components/About/About.js

```diff
@@ -2,7 +2,6 @@ import React, { Component } from 'react';
 import '../../styles/About.css';
 import Blurbs from "./Blurbs.js";
```

Comment:
About.js: Good to import Blurbs.js in OurStory.js, import OurStory.css in OurStory.js, and import OurStory.js in About.js to ease future additions/amendments.